test/common/update_instance.go

```diff
@@ -40,8 +40,10 @@ func TestUpdateInstance(
 
 		if async {
 			Convey("should return 422 UnprocessableEntity if missing accepts_incomplete", func() {
+				tempBody := openapi.ServiceInstanceUpdateRequest{}
+				deepCopy(req, &tempBody)
```

Comment:
What's the point of this change?

Reply:
When provisioning an instance it is required to have service_id and plan_id to be non-empty values. This test case was missing this, which caused it to fail no matter what. This has been fixed now.